src/utils/source.js

```diff
@@ -329,3 +339,12 @@ export function getTextAtPosition(source: Source, location: Location) {
 
   return lineText.slice(column, column + 100).trim();
 }
+
+export function getSourceClassnames(source: Object) {
+  if (source && source.isBlackBoxed) {
+    return "blackBox";
+  }
+  const sourceType = sourceTypes[getExtension(source)];
+  const classNames = classnames(sourceType || "file");
+  return classNames;
```

Comment:
do we need `classnames(sourceType || "file")`

could it just be `return sourceType || "file"`
`